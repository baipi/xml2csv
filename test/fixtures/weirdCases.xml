<Weird>
  <Case>
    <First>This sentence has "double quotes"</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>This sentence has , comma</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>4"2</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>4,2</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>"42</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>,42</First>
    <Second>control</Second>
  </Case>
  <Case>
    <First>42</First>
    <Second>control</Second>
  </Case>
  <Case>
    <Second>control</Second>
  </Case>
  <Case>
    <First>42</First>
  </Case>
  <Case>
  </Case>
</Weird>