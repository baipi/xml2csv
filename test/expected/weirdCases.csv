first,second
"This sentence has ""double quotes""","control"
"This sentence has , comma","control"
"4""2","control"
"4,2","control"
"""42","control"
",42","control"
"42","control"
,"control"
"42",
,
